src/test/java/lotto/domain/LottoLotteryTest.java

```diff
@@ -2,16 +2,32 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+import java.util.Arrays;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
 class LottoLotteryTest {
     @Test
     @DisplayName("구매한 복권의 갯수만큼 자동 로또가 생성된다")
-    void test() {
+    void create() {
         LottoPurchaseAmount lottoPurchaseAmount = new LottoPurchaseAmount("3000");
         LottoPurchaseQuantity lottoPurchaseQuantity = LottoPurchaseQuantity.of(lottoPurchaseAmount.calculateQuantity());
         LottoLottery lottoLottery = LottoLottery.of(lottoPurchaseQuantity, new AutoNumberGenerator());
         assertThat(lottoLottery).isInstanceOf(LottoLottery.class);
```

Comment:
```suggestion
        LottoPurchaseAmount lottoPurchaseAmount = new LottoPurchaseAmount("3000");
        LottoPurchaseQuantity lottoPurchaseQuantity = LottoPurchaseQuantity.of(lottoPurchaseAmount.calculateQuantity());
        
        LottoLottery lottoLottery = LottoLottery.of(lottoPurchaseQuantity, new AutoNumberGenerator());
        
        assertThat(lottoLottery).isInstanceOf(LottoLottery.class);
```

[Given-When-Then](https://kchanguk.tistory.com/40)
블로그 내용처럼 given,when,then을 주석으로 작성할 수도 있지만, 
개행 처리로도 테스트 코드의 흐름을 쉽게 파악할 수 있을것 같아요 !